fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -750,16 +750,18 @@ public void finishTransaction(long transactionId, Set<Long> errorReplicaIds) thr
     
     // check if there exists a load job before the endTransactionId have all finished
     // load job maybe started but could not know the affected table id, so that we not check by table
-    public boolean isPreviousTransactionsFinished(long endTransactionId, long dbId) {
+    public boolean isPreviousTransactionsFinished(long endTransactionId, long dbId, boolean printLog) {
```

Comment:
I think it is not reasonable to add a parameter to control if it print log.

Reply:
change log level to debug